fileop.go

```diff
@@ -40,6 +40,18 @@ func Delete(fileName string) error {
 	return nil
 }
 
+func FileExists(fileName string) (bool, error) {
+	var wfd wrappers.WIN32_FIND_DATA
+	handle, err := wrappers.FindFirstFile(syscall.StringToUTF16Ptr(fileName), &wfd)
```

Comment:
That's not going to work because we're using this to check a wildcard.  Also, even if it would, couldn't using `GetFileAttributes` cause problems in cases where we had `FILE_LIST_DIRECTORY` access to the directory but not `FILE_READ_ATTRIBUTES` access to (some of) the files in it?